src/main/pg/pg_ids.h

```diff
@@ -152,7 +152,8 @@
 #define PG_MODE_ACTIVATION_CONFIG 553
 #define PG_DYN_NOTCH_CONFIG 554
 #define PG_RX_EXPRESSLRS_SPI_CONFIG 555
-#define PG_BETAFLIGHT_END 555
+#define PG_SCHEDULER_CONFIG 536
+#define PG_BETAFLIGHT_END 556
```

Comment:
```suggestion
#define PG_BETAFLIGHT_END 557
```

Reply:
PG_BETAFLIGHT_END is equal to the last entry, not one greater. In master currently it's.

```
#define PG_RX_EXPRESSLRS_SPI_CONFIG 555
#define PG_BETAFLIGHT_END 555
```